src/i18n.ts

```diff
@@ -7,16 +8,24 @@ const resources = {
     translation: resourcesPL
   },
   en: {
-    translation: resourcesPL
+    translation: resourcesEN
   }
 };
 
 i18n.use(initReactI18next).init({
-  resources,
   lng: "pl",
   interpolation: {
     escapeValue: false
-  }
+  },
+  resources:{
+    pl:{
```

Comment:
Language resources are already defined above. WHY THE FUCK ARE THEY REDEFINED HERE?!